The panic hook replaces a helpful backtrace by a generic and largely
unhelpful message - disable that.
In addition, disable the generation of a stacktrace URL, and whatever the tunnel
feature is supposed to do.

Index: cli/main.rs
--- cli/main.rs.orig
+++ cli/main.rs
@@ -147,9 +147,10 @@ async fn run_subcommand(
       );
       tools::bundle::bundle(flags, bundle_flags).await
     }),
-    DenoSubcommand::Deploy(deploy_flags) => spawn_subcommand(async move {
-      tools::deploy::deploy(Arc::unwrap_or_clone(flags), deploy_flags).await
-    }),
+    DenoSubcommand::Deploy(deploy_flags) => exit_with_message(
+      "Deno Deploy is currently not supported on OpenBSD.",
+      1,
+    ),
     DenoSubcommand::Doc(doc_flags) => {
       spawn_subcommand(async { tools::doc::doc(flags, doc_flags).await })
     }
@@ -610,7 +611,7 @@ pub fn main() {
   #[cfg(feature = "dhat-heap")]
   let profiler = dhat::Profiler::new_heap();
 
-  setup_panic_hook();
+  // setup_panic_hook();
 
   init_logging(None, None);
 
@@ -729,11 +730,13 @@ async fn resolve_flags_and_init(
     .map(|files| files.iter().map(PathBuf::from).collect());
   load_env_variables_from_env_files(env_file_paths.as_ref(), flags.log_level);
 
+  #[cfg(not(target_os="openbsd"))]
   if deno_lib::args::has_flag_env_var("DENO_CONNECTED") {
     flags.tunnel = true;
   }
 
   // Tunnel sets up env vars and OTEL, so connect before everything else.
+  #[cfg(not(target_os="openbsd"))]
   if flags.tunnel && !matches!(flags.subcommand, DenoSubcommand::Deploy(_)) {
     if let Err(err) = initialize_tunnel(&flags).await {
       exit_for_error(
@@ -747,6 +750,12 @@ async fn resolve_flags_and_init(
     }
   }
 
+  #[cfg(target_os="openbsd")]
+  if flags.tunnel {
+    exit_with_message("--connected and --tunnel are not supported on OpenBSD", 1);
+  }
+  
+
   flags.unstable_config.fill_with_env();
   if std::env::var("DENO_COMPAT").is_ok() {
     flags.unstable_config.enable_node_compat();
@@ -991,6 +1000,7 @@ struct AuthTunnelOutput {
   token: String,
 }
 
+#[cfg(not(target_os="openbsd"))]
 async fn auth_tunnel(
   no_config: bool,
   env_token: Option<String>,
@@ -1026,6 +1036,7 @@ async fn auth_tunnel(
 }
 
 #[allow(clippy::print_stderr)]
+#[cfg(not(target_os="openbsd"))]
 async fn initialize_tunnel(
   flags: &Flags,
 ) -> Result<(), deno_core::anyhow::Error> {
